Providers/Profile.swift

```diff
@@ -291,6 +297,17 @@ public class BrowserProfile: Profile {
         return self.remoteClientsAndTabs.getClientsAndTabs()
     }
 
+
+    public func sendItems(items: [ShareItem], toClients clients: [RemoteClient]) {
+        var cmds = [SyncCommand]()
+        for client in clients {
+            for item in items {
+                cmds.append(SyncCommand(guid: Bytes.generateGUID(), clientGuid: client.guid!, url: item.url, title: item.title, faviconID: nil, action: SentTabAction.View.rawValue, lastUsed: NSDate.now()))
```

Comment:
Commands don't have a place for a GUID, so let's just kill that field and save a little effort.

s/clientGuid/clientGUID/

and as noted above, let's lift this into a factory method on `SyncCommand`.